src/controllers/userController.js

```diff
@@ -82,5 +82,16 @@ const User = {
     });
   },
 
+  async updateProfile(req, res) {
+    const { email, role, ...updateData } = {
+      ...req.body, avatar: req.imgLink, password: helpers.hashPassword(req.body.password)
+    };
+    const user = await db.Users.update(updateData, {
+      where: { email: req.user.email }, returning: true, plain: true
+    });
+    const { password, ...data } = user[1].dataValues;
```

Comment:
Nice big work @swaibat
you can just use `raw: true` instead of `plain: true` it will return an array, after this
on line 92:  you will have to just write `const { password, ...data } = user[1]`
use the same procedure in, `src/middlewares/userMiddlware.js: line 32`
one more thing: `do it if, you find it necessary`
Good job

Reply:
@urbanishimwe  they are all the same because it also returns an array `const { password, ...data } = user[1]` if it wasn't an array you cant access it by index like this `user[1]`

Reply:
well, they're not because you're doing `user[1].dataValues` not `user[1]`